packages/api/src/command/medical/patient/__tests__/consolidated-get.test.ts

```diff
@@ -53,3 +56,30 @@ describe("getConsolidatedPatientData", () => {
     expect(fhir_searchResourcePages).toHaveBeenCalledTimes(1);
   });
 });
+
+describe("getCurrentConsolidatedProgress", () => {
```

Comment:
can we add tests for when some of the request params are undefined to confirm the behavior of getCurrentConsolidatedProgress? 

Some of these have default behaviors that would make some of the checks in getCurrentConsolidatedProgress fail but would make two requests actually the same. 

For example, one request specifies conversion type of json. The other request doesnt. But json is the default. So they are the same actually but the `getCurrentConsolidatedProgress` wont catch that. Same thing for default of resources being all resources and specifying all resources being the same. 

So lets add tests for those cases and then modify the test function accordingly?

Reply:
json i made as a default but added tests for the resources